sys-auth/pam_u2f/pam_u2f-1.1.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools flag-o-matic pam
+
+DESCRIPTION="PAM module for FIDO2 and U2F keys"
+HOMEPAGE="https://github.com/Yubico/pam-u2f"
+SRC_URI="https://developers.yubico.com/${PN/_/-}/Releases/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+RDEPEND="
+	dev-libs/libfido2
```

Comment:
Per https://github.com/Yubico/pam-u2f/compare/pam_u2f-1.0.8...pam_u2f-1.1.0#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810R59, you're missing a dependency on libcrypto (openssl or libressl). It's better to make users of both libraries happy by using something like the following:

```
!libressl? ( dev-libs/openssl:0= )
libressl? ( dev-libs/libressl:0= )
```